etc/latex-common.profile

```diff
@@ -0,0 +1,38 @@
+# Firejail profile for latex-common
+# This file is overwritten after every install/update
+# Persistent local customizations
+include latex-common.local
+# Persistent global definitions
+# added by caller profile
```

Comment:
Sorry to bother you again haha. After this line we add

#include globals.local

to remember that we need globals.local, but make an exception here because this is the common file.

Reply:
So just add  ```#include globals.local``` in latex-common.profile on line 7 and be done with it?

Reply:
I'm asking because this is inconsistent in other common profiles:
```
$ find -xdev -name '*common*' -exec sh -c 'echo "filename: {}"; grep --with-filename "globals.local" "{}"' \;
filename: ./transmission-common.profile
filename: ./discord-common.profile
./discord-common.profile:#include globals.local
filename: ./autokey-common.profile
./autokey-common.profile:#include globals.local
filename: ./whitelist-var-common.inc
filename: ./whitelist-common.inc
filename: ./eo-common.profile
./eo-common.profile:#include globals.local
filename: ./latex-common.profile
filename: ./allow-common-devel.inc
filename: ./firefox-common.profile
./firefox-common.profile:#include globals.local
filename: ./whitelist-usr-share-common.inc
filename: ./chromium-common.profile
./chromium-common.profile:#include globals.local
filename: ./firefox-common-addons.inc
filename: ./disable-common.inc
```

Reply:
> So just add #include globals.local in latex-common.profile on line 7 and be done with it?

Correct. After you push this last change I'll have another quick look but I think this is ready for merging.

> I'm asking because this is inconsistent in other common profiles:

Indeed it is! You just found a few omissions. I'll take care of those later. Thank you for checking that, we always seem to be short of extra eyes :+1: .

Reply:
> > So just add #include globals.local in latex-common.profile on line 7 and be done with it?
> 
> Correct. After you push this last change I'll have another quick look but I think this is ready for merging.
> 
> > I'm asking because this is inconsistent in other common profiles:
> 
> Indeed it is! You just found a few omissions. I'll take care of those later. Thank you for checking that, we always seem to be short of extra eyes +1 .

Glad to help!

Reply:
Ah, those `.inc` files are special too, although in another context. Those never need to reference globals.local. The only ommission that actually needs fixing is `transmission-common.profile`. As you found it, credit should be yours. If you wish, you can prepare another PR for that and it can be merged really quick. Just add the below lines to transmission-common.profile directly after 'include transmission-common.local':


```
# Persistent global definitions
# added by caller profile
#include globals.local
```